src/Tooltip.js

```diff
@@ -34,6 +34,7 @@ const propTypes = {
     PropTypes.string,
     PropTypes.number
   ]),
+  innerRef: PropTypes.object,
```

Comment:
Technically this can still be a string or a function as well as the new createRef object.

Also need to document the prop (at the very least add it to the props on the docs for this component)

Reply:
You're right. I've changed the propType definition and added the proper explanation in the docs